### PR TITLE
fix: stashing changes before pull in ./setup.sh

### LINKED ISSUE #8: failed execution of ./setup.sh in second run.
So if running ./setup.sh doesn't work in first time (my case), so running the setup.sh script fails to run, as a package.lock.json is created and in the script it pulls the changes and you cannot pull without committing changes.

image

### PR DESCRIPTION
**Pull request by anuragts**
fixes #8 

Stashing changes before pulling changes.

And poping the stash after git pull.

### CODE REVIEW

**Comment by sankalp1999** [comment]
thanks for the PR. I think people can manage this themselves

File: setup.sh [hunk 1]
```diff
@@ -8,8 +8,14 @@ set -e
...
 # Run each JavaScript file in synchronous order
 
+echo "You have uncommited changes, stashing the changes to proceed with the pull"
+git stash
+
 git pull
 
+echo "Popping the stash to reapply the changes"
+git stash pop
+
```
Comment (sankalp1999) [inline setup.sh]: this may cause merge conflict and stop the script from running. that's my main concern.

Comment (anuragts) [reply setup.sh]: hmm yeah, maybe only stash changes and echo that changes been stashed

Comment (anuragts) [reply setup.sh]: removes stash pop

**Review by sankalp1999 — APPROVED** [review]

**Comment by sankalp1999** [comment]
thanks for the help!

**Pull request merged**